### PR TITLE
Selenium page object

### PR DESCRIPTION
**Pull request by kanagalakshmi74**
Implemented few changes except utility wait class. Changed the class file as Pascal Naming but not reflected in branch.

### CODE REVIEW

File: LoginDemo/LearningSelenium/PageModel/product.cs [hunk 1]
```diff
@@ -7,40 +7,39 @@
...
+        public By backMenu = By.XPath("//button[contains(@id,'back-to-products')]");
+        private By item2 = By.PartialLinkText("Backpack");
+        private By addItem2 = By.XPath("//button[@id='add-to-cart-sauce-labs-backpack' or @id='remove-sauce-labs-backpack']");
+        private By itemCount = By.XPath("//a[contains(@class,'shopping_cart_link')]/span[contains(@class,'shopping_cart_badge')]");
         public Product(WebDriver driver)
         {
             this.driver = driver;
         }
-        public void getItem1()
+        public void GetItem1()
```
Comment (rudzkokseniya) [inline LoginDemo/LearningSelenium/PageModel/product.cs:23]: You can combine similar methods into one and pass item as parameter

File: LoginDemo/LearningSelenium/PageModel/product.cs [hunk 2]
```diff
@@ -7,40 +7,39 @@
...
         {
-            driver.FindElement(Item1).Click();
+            driver.FindElement(item1).Click();
         }
-        public void getItem2()
+        public void GetItem2()
```
Comment (rudzkokseniya) [inline LoginDemo/LearningSelenium/PageModel/product.cs:27]: One empty line between methods

File: LoginDemo/LearningSelenium/Shopping.cs [hunk 3]
```diff
@@ -26,109 +26,109 @@ public void Setup()
...
-            lg.enterSwagUserName("standard_user");
-            lg.enterSwagPassword("secret_sauce");
-            lg.SwagLoginClick();
+            login.EnterSwagUserName("standard_user");
+            login.EnterSwagPassword("secret_sauce");
+            login.SwagLoginClick();
             WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(20));
-            wait.Until(d => lg.getSwagSuccessMessage());
-            Assert.AreEqual("Products", lg.getSwagSuccessMessage(), "loged user name is mismatch");
+            wait.Until(d => login.GetSwagSuccessMessage());
```
Comment (rudzkokseniya) [inline LoginDemo/LearningSelenium/Shopping.cs:83]: Use more meaningful variables names

File: LoginDemo/LearningSelenium/PageModel/product.cs (hunk 2 above)
Comment (kanagalakshmi74) [reply LoginDemo/LearningSelenium/PageModel/product.cs:27]: ok will give empty line between methods

File: LoginDemo/LearningSelenium/Shopping.cs (hunk 3 above)
Comment (kanagalakshmi74) [reply LoginDemo/LearningSelenium/Shopping.cs:83]: variable d is changed

**Pull request merged**